spec/controllers/api/v1/packages_controller_spec.rb

```diff
@@ -1596,41 +1611,207 @@ def unpack(qty, pkg, out_of:)
           task: 'pack',
           quantity: 5
         }
+
         @params7 = {
           id: box.id,
           item_id: package2.id,
           location_id: location.id,
           task: "pack",
           quantity: package2.on_hand_quantity + 20,
         }
+
```

Comment:
I think we should be using `let` over instance variables for tests.

Also I think it's better to keep a single param sample that you can override in your tests. Because we don't know where each param is used for.

Or if you prefer having multiple params, maybe they should be named explicitly, e.g `let(:add_package_to_pallet_params) { ... }`

note: I know this pre-existed your changes, just pointing it out :)

Reply:
done. refactored it : )